Dpkg building from source ends with a "dpkg failed to install: `make V=0 -j2` exited with 2"

I executed
```shell
crew install dpkg
```

Output:
```
  CC       update.o
  CC       verify.o
  CC       divertcmd.o
  CC       querycmd.o
  CC       statcmd.o
  CC       trigcmd.o
  CCLD     dpkg
  CCLD     dpkg-divert
  CCLD     dpkg-query
  CCLD     dpkg-statoverride
  CCLD     dpkg-trigger
make[2]: Leaving directory '/usr/local/tmp/crew/dpkg-1.19.0.5.tar.gz.dir/dpkg-1.19.0.5/dpkg/src'
Making all in utils
make[2]: Entering directory '/usr/local/tmp/crew/dpkg-1.19.0.5.tar.gz.dir/dpkg-1.19.0.5/dpkg/utils'
  CC       start-stop-daemon.o
  CC       update_alternatives-update-alternatives.o
  GEN      org.dpkg.pkexec.update-alternatives.policy
  CCLD     update-alternatives
  CCLD     start-stop-daemon
make[2]: Leaving directory '/usr/local/tmp/crew/dpkg-1.19.0.5.tar.gz.dir/dpkg-1.19.0.5/dpkg/utils'
Making all in dselect
make[2]: Entering directory '/usr/local/tmp/crew/dpkg-1.19.0.5.tar.gz.dir/dpkg-1.19.0.5/dpkg/dselect'
Making all in methods
make[3]: Entering directory '/usr/local/tmp/crew/dpkg-1.19.0.5.tar.gz.dir/dpkg-1.19.0.5/dpkg/dselect/methods'
  GEN      ftp/update
  GEN      ftp/setup
  GEN      ftp/install
make[3]: Leaving directory '/usr/local/tmp/crew/dpkg-1.19.0.5.tar.gz.dir/dpkg-1.19.0.5/dpkg/dselect/methods'
Making all in po
make[3]: Entering directory '/usr/local/tmp/crew/dpkg-1.19.0.5.tar.gz.dir/dpkg-1.19.0.5/dpkg/dselect/po'
make[3]: *** No rule to make target 'all'.  Stop.
make[3]: Leaving directory '/usr/local/tmp/crew/dpkg-1.19.0.5.tar.gz.dir/dpkg-1.19.0.5/dpkg/dselect/po'
make[2]: *** [Makefile:642: all-recursive] Error 1
make[2]: Leaving directory '/usr/local/tmp/crew/dpkg-1.19.0.5.tar.gz.dir/dpkg-1.19.0.5/dpkg/dselect'
make[1]: *** [Makefile:659: all-recursive] Error 1
make[1]: Leaving directory '/usr/local/tmp/crew/dpkg-1.19.0.5.tar.gz.dir/dpkg-1.19.0.5/dpkg'
make: *** [Makefile:567: all] Error 2  CC       update.o
  CC       verify.o
  CC       divertcmd.o
  CC       querycmd.o
  CC       statcmd.o
  CC       trigcmd.o
  CCLD     dpkg
  CCLD     dpkg-divert
  CCLD     dpkg-query
  CCLD     dpkg-statoverride
  CCLD     dpkg-trigger
make[2]: Leaving directory '/usr/local/tmp/crew/dpkg-1.19.0.5.tar.gz.dir/dpkg-1.19.0.5/dpkg/src'
Making all in utils
make[2]: Entering directory '/usr/local/tmp/crew/dpkg-1.19.0.5.tar.gz.dir/dpkg-1.19.0.5/dpkg/utils'
  CC       start-stop-daemon.o
  CC       update_alternatives-update-alternatives.o
  GEN      org.dpkg.pkexec.update-alternatives.policy
  CCLD     update-alternatives
  CCLD     start-stop-daemon
make[2]: Leaving directory '/usr/local/tmp/crew/dpkg-1.19.0.5.tar.gz.dir/dpkg-1.19.0.5/dpkg/utils'
Making all in dselect
make[2]: Entering directory '/usr/local/tmp/crew/dpkg-1.19.0.5.tar.gz.dir/dpkg-1.19.0.5/dpkg/dselect'
Making all in methods
make[3]: Entering directory '/usr/local/tmp/crew/dpkg-1.19.0.5.tar.gz.dir/dpkg-1.19.0.5/dpkg/dselect/methods'
  GEN      ftp/update
  GEN      ftp/setup
  GEN      ftp/install
make[3]: Leaving directory '/usr/local/tmp/crew/dpkg-1.19.0.5.tar.gz.dir/dpkg-1.19.0.5/dpkg/dselect/methods'
Making all in po
make[3]: Entering directory '/usr/local/tmp/crew/dpkg-1.19.0.5.tar.gz.dir/dpkg-1.19.0.5/dpkg/dselect/po'
make[3]: *** No rule to make target 'all'.  Stop.
make[3]: Leaving directory '/usr/local/tmp/crew/dpkg-1.19.0.5.tar.gz.dir/dpkg-1.19.0.5/dpkg/dselect/po'
make[2]: *** [Makefile:642: all-recursive] Error 1
make[2]: Leaving directory '/usr/local/tmp/crew/dpkg-1.19.0.5.tar.gz.dir/dpkg-1.19.0.5/dpkg/dselect'
make[1]: *** [Makefile:659: all-recursive] Error 1
make[1]: Leaving directory '/usr/local/tmp/crew/dpkg-1.19.0.5.tar.gz.dir/dpkg-1.19.0.5/dpkg'
make: *** [Makefile:567: all] Error 2
```
Chromebrew 0.4.3.

ChromeOS: `CHROMEOS_ARC_ANDROID_SDK_VERSION=25
CHROMEOS_ARC_VERSION=4898482
CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={7834CC99-3294-421F-9963-028D7D6512CA}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={7834CC99-3294-421F-9963-028D7D6512CA}
CHROMEOS_RELEASE_BOARD=celes-signed-mp-v2keys
CHROMEOS_RELEASE_BRANCH_NUMBER=0
CHROMEOS_RELEASE_BUILDER_PATH=celes-release/R69-10888.0.0
CHROMEOS_RELEASE_BUILD_NUMBER=10888
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=69
CHROMEOS_RELEASE_DESCRIPTION=10888.0.0 (Official Build) dev-channel celes 
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=dev-channel
CHROMEOS_RELEASE_VERSION=10888.0.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=10888.0.0`

Model: Celes

Architecture: amd64

It's just a problem with make, I've only seen this error on https://github.com/skycocker/chromebrew/issues/1406, but it's not related to this (at least I don't think so).

make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/usr/local/tmp/dpkg/utils'
Making all in dselect
make[2]: Entering directory '/usr/local/tmp/dpkg/dselect'
Making all in methods
make[3]: Entering directory '/usr/local/tmp/dpkg/dselect/methods'
make[3]: Nothing to be done for 'all'.
make[3]: Leaving directory '/usr/local/tmp/dpkg/dselect/methods'
Making all in po
make[3]: Entering directory '/usr/local/tmp/dpkg/dselect/po'
make[3]: *** No rule to make target 'all'.  Stop.
make[3]: Leaving directory '/usr/local/tmp/dpkg/dselect/po'
make[2]: *** [Makefile:642: all-recursive] Error 1
make[2]: Leaving directory '/usr/local/tmp/dpkg/dselect'
make[1]: *** [Makefile:659: all-recursive] Error 1
make[1]: Leaving directory '/usr/local/tmp/dpkg'
make: *** [Makefile:567: all] Error 2

--- a/packages/dpkg.rb
+++ b/packages/dpkg.rb
@@ -3,17 +3,16 @@ require 'package'
 class Dpkg < Package
   description 'A medium-level package manager for Debian'
   homepage 'https://anonscm.debian.org/git/dpkg/'
-  version '1.19.0.5-2'
-  source_url 'https://salsa.debian.org/dpkg-team/dpkg/-/archive/1.19.0.5/dpkg-1.19.0.5.tar.gz'
-  source_sha256 'd38308afcd5d7896bbd1f946875b90f9d8510a8a96b44e4f14e781285e5d9641'
+  version '1.18.25'
+  source_url 'https://salsa.debian.org/dpkg-team/dpkg/-/archive/1.18.25/dpkg-1.18.25.tar.gz'
+  source_sha256 '93470770161bb15fd7988da1890aecda322e19c3fc858d6cb8f33417e5fe01e9'
 
   depends_on 'bz2'
   depends_on 'xzutils'
 
   def self.build
-    system "git clone https://salsa.debian.org/dpkg-team/dpkg.git"
+    system "git clone https://salsa.debian.org/dpkg-team/dpkg.git -b 1.18.25"
     Dir.chdir ("dpkg") do
-      system "git checkout 1.19.0.5"
       system "autoreconf -i -f"
       system "./configure --libdir=#{CREW_LIB_PREFIX} --prefix=#{CREW_PREFIX}"
       system "make"

